QUICKSTART.md

```diff
@@ -0,0 +1,45 @@
+# Kind
+This guide was created to help new users of Kubeflow install and test Kubeflow Helm Charts locally based on the kind.
+
+## Requirements
+* kind v0.25.0 or above
+* helm v3.16.3 or above
```

Comment:
`v3.16.3` is just 3 weeks old. I'm using `v3.15.4` on my local machine and it works fine. Maybe we can write "tested on `v3.15.4` but should work with older versions as well"?